libs/ui/src/admin/ArtApprovalModal/ArtApprovalForm.tsx

```diff
@@ -0,0 +1,113 @@
+import { FC, memo } from 'react'
+
+import {
+  Avatar,
+  HStack,
+  Text,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalOverlay,
+  Stack,
+  Box,
+  Image,
+  Center,
+  Flex,
+  SimpleGrid,
+} from '@chakra-ui/react'
+import { Splide, SplideSlide } from '@splidejs/react-splide'
+
+import { ArtFeedbackForm } from './ArtFeedbackForm'
+import { ArtImageProps, ArtApprovalFormTypes } from './types'
+
+const ArtImage: FC<ArtImageProps> = memo(({ image, alt }) => {
+  return (
+    <Image
+      objectFit="contain"
+      src={`${process.env['NX_API_URL']}${image.url}`}
+      alt={alt}
+    />
+  )
+})
+
+export const ArtApprovalForm: FC<ArtApprovalFormTypes> = ({
+  onReject,
+  onApprove,
+  onDelete,
+  artId,
+  artDescription,
+  artTitle,
+  artAvatar,
+  artImages,
+  editorId,
+  isOpen,
+  onClose,
+  editorAvatar,
+  editorName,
+  artArtistName,
+  isEdit,
+}) => {
+  return (
+    <Box>
+      <Modal onClose={onClose} size={['1088px']} isOpen={isOpen}>

```

Comment:
```suggestion
      <Modal onClose={onClose} size="6xl" isOpen={isOpen}>
```

I'm not sure we can give size as pixels.